Orm/Xtensive.Orm/Orm/Linq/Translator.Expressions.cs

```diff
@@ -646,6 +654,9 @@ internal static bool FilterBindings(MemberInfo mi, string name, Type type)
       return bindings;
     }
 
+    private static ConstantExpression SelectBoolConstantExpression(bool b) =>
+      b ? trueBoolExpression : falseBoolExpression;
+
```

Comment:
Place functions in the downward direction.

Reply:
done